app/views/managements/dashboard.html.erb

```diff
@@ -87,7 +87,7 @@
               <tr>
                 <td><a href="/slides/<%= slide.id %>"><%= slide.id %></a></td>
                 <td><%= slide.name %></td>
-                <td><%= slide.user_display_name %></td>
+                <td><%= link_to slide.user_display_name, "/#{slide.user.username}" %></td>
```

Comment:
__[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)